source/license-openssl-ssleay.txt

```diff
@@ -10,7 +10,7 @@
   ---------------
 
 /* ====================================================================
- * Copyright (c) 1998-2018 The OpenSSL Project.  All rights reserved.
+ * Copyright (c) 1998-2019 The OpenSSL Project.  All rights reserved.
```

Comment:
I don't think this is correct.  The SSLeay copyright date should stay frozen as of the change to CLA's and such.  Consult a lawyer.  No SSLeay-copyright code was changed; code *derived* from it is, but that is covered by the Apache license now and was always covered by CLA terms starting back in the day.

Reply:
Huh? I'm not touching the SSLeay copyright date. This is the OpenSSL licence. Code covered by the OpenSSL licence has changed in 2019 (i.e. all activity in the 1.0.2, 1.1.0 and 1.1.1 stable branches).

Reply:
But *this file*, ```license-openssl-ssleay.txt``` hasn't changed.  The license text hasn't changed. Files that refer to the license, which all have specify copyright years for those files, and looking at (for example) the 1.02 releases, each file has the license text. In 1.1.0, each file points to the LICENSE file.

There really should be no reason to change the copyright on this file which hasn't been modified in over a year.  You don't need to change the copyright on the license if the license itself hasn't changed.

Reply:
This file hadn't changed last year either...  and yet, we updated the year range to include 2018 at the time.  If you look at the history of `LICENSE` in the source up to 1.1.1, you'll see that this is fairly routine (and approved by you more than once).

Reply:
Either way, this file is a copy of the `LICENSE` file in the latest release (currently 1.1.1c), so if it changed there, it changes here.

Reply:
This is the licence that applies to all the code in the stable branches. It is not *just* the licence for this file itself....otherwise the file would be fairly pointless. To try and claim that the scope of the license is for all of the code but the scope of the copyright claim is just for this file is pretty weird and not at all what I think people looking at this file will expect or understand that copyright claim to mean. It certainly is not my interpretation of what it means and my guess is you would be hard pressed to find new people coming to the project who would interpret it in the way you suggest it should be.

Reply:
Yes I approved previous changes; I was wrong.